pkg/systemd/overview-cards/loginMessage.jsx

```diff
@@ -0,0 +1,74 @@
+/*
```

Comment:
Could files `pkg/systemd/overview-cards/loginMessage.jsx` and `pkg/systemd/overview-cards/lastLogin.jsx ` be unified, or is there a good reason why they are two separate files?